Documentation/CRDs/Cluster/external-cluster/external-cluster.md

```diff
@@ -114,6 +114,16 @@ Restricted users created using `--restricted-auth-permission` flag need to pass
     An existing non-restricted user cannot be converted to a restricted user by upgrading.
     The upgrade flag should only be used to append new permissions to users. It shouldn't be used for changing a csi user already applied permissions. For example, you shouldn't change the pool(s) a user has access to.
 
+#### Upgrade and utilize new feature
```

Comment:
Something weird I notice is that the documentation steps for upgrades come in between the 2 steps `### 1. Create all users and keys` and `### 2. Copy the bash output`. I thin k the upgrade section should be placed at the end since it is not something users need to consider when first interacting with this doc.